### PR TITLE
DESCW-1536 updated nginx to have access to plugin/themes, and added c…

### PR DESCRIPTION
**Pull request by ShawnTurple**
…onditional logic to require in config

### CODE REVIEW

**Review by robrien-bcgov — APPROVED** [review]
seems reasonable

File: deployments/kustomize/base/wordpress/deployment.yaml [hunk 1]
```diff
@@ -89,6 +89,14 @@ spec:
         - name: nginx-configmap-volume
           mountPath: /etc/nginx/conf.d/nginx.conf
           subPath: nginx.conf
+        - name: plugins-themes
+          mountPath: /var/www/html/wp-content/plugins
+          subPath: wp-content/plugins
+          readOnly: true
+        - name: plugins-themes
```
Comment (mhaswell-bcgov) [inline deployments/kustomize/base/wordpress/deployment.yaml:96]: These have the same name, not sure if that's intentional.

Comment (kshapka-bcgov) [reply deployments/kustomize/base/wordpress/deployment.yaml:96]: It's using the same volume (plugins-themes) twice, once for plugins and once for themes.

Comment (ShawnTurple) [reply deployments/kustomize/base/wordpress/deployment.yaml:96]: It references the volume, just different mount paths.

**Pull request merged**